conf/application.conf

```diff
@@ -66,12 +66,24 @@ mongodb {
   uri = "mongodb://localhost:27017/pega-proof-of-concept-proxy"
 }
 
+pega-proof-of-concept-stubs-uris {
+    submit-payload = "/pega-proof-of-concept-stubs/submit-payload"
+}
+
+pega-proof-of-concept-stubs {
+    baseUrl = "http://localhost:9254"
```

Comment:
do you still need this? You can derive it from the `miroservices.services` part of the config below using `ServicesConfig`- you'd just need one or the other I think